controller/internal/routes/identity_api_model.go

```diff
@@ -317,3 +317,46 @@ type IdentityServiceConfig struct {
 func (entity IdentityServiceConfig) toModel() model.ServiceConfig {
 	return model.ServiceConfig{Config: entity.Config, Service: entity.Service}
 }
+
+type AdvisorEdgeRouter struct {
+	*EntityApiRef
+	Online bool `json:"isOnline"`
```

Comment:
I am partial to `IsOnline` for the struct property name as well.

Reply:
Fixed